inc/Engine/CDN/Subscriber.php

```diff
@@ -118,7 +112,7 @@ public function rewrite_css_properties( $content ) {
 			return $content;
 		}
 
-		if ( ! $this->options->get( 'cdn' ) ) {
+		if ( ! $this->options->get( 'cdn', 0 ) ) {
```

Comment:
This pattern repeats in the class. Let's move it to a method to encapsulate the knowledge of how to get the `'cdn'` option and its default.

Reply:
https://github.com/wp-media/wp-rocket/pull/2589/commits/0143b88bf6ffa55847b18305f7e4ce1ad0e409ac